jsapp/js/components/library/singleCollectionStore.ts

```diff
@@ -24,6 +24,7 @@ import type {AssetTypeName} from 'js/constants';
 
 interface SingleCollectionStoreData {
   isFetchingData: boolean;
+  isTranscriptInProgress: boolean;
```

Comment:
This shouldn't even be in this file. I see @jnm added this line and the other one with `isTranscriptInProgress`. Both of these should be removed.

Reply:
Right, that's junk that I added and should've removed after changing it to `isPollingForTranscript`